nClam/ClamScanInfectedFile.cs

```diff
@@ -3,16 +3,22 @@
     /// <summary>
     /// The results of an infected file.
     /// </summary>
-    public class ClamScanInfectedFile
+    public record ClamScanInfectedFile
```

Comment:
This could be simplified to `public record ClamScanInfectedFile(string FileName, string VirusName);`

Reply:
I tried doing this, but it would not let me target for `netstandard2.0` anymore.  I tried `netstandard2.1` and that did not work.  If I target `net5.0` it works - so this may be a future change we can incorporate if we bump the target framework.  The current library version has 80k downloads, so I rather not change target framework if at all possible.